website/docs/docs/build/python-models.md

```diff
@@ -0,0 +1,649 @@
+---
+title: "Python models"
+id: "python-models"
+---
+- [Overview](#overview)
+- [Configuring Python Models](#configuring-python-models)
+- [Python Specific Functionality](#configuring-python-models)
+- [Limitations](#limitations)
+- [Supported data platforms](#supported-data-platforms)
+
+:::info
+
+dbt Core v1.3 adds support for Python models. Note that only [specific data platforms](#specific-data-platforms) support dbt-py models.
+
+We encourage you to:
```

Comment:
Can we move this out of the note? It's a huge and hard-to-read this way. 

I also suggests versioning this paragraph for 1.3 so it's less confusing:
> In the following article, you'll see a section titled "❓ dbt questions." We are excited to release the first set of functionality in v1.3, which will solve real use cases. We also know this is the first step toward a much wider field of possibility. We don't pretend to have all the answers.

Reply:
YES!  Thank you for that feedback.  I wasn't a fan.